### PR TITLE
Pivo payment parameters,

### PR DESCRIPTION
**Pull request by Kostersson**
Fix one 2017 expired test card

### CODE REVIEW

**Comment by jarikujansuu** [comment]
Tests fail. `FormAPIConnectionTest.testGetFormSubmitResponse`

File: src/main/java/io/paymenthighway/FormBuilder.java [hunk 1]
```diff
@@ -188,6 +188,41 @@ public MasterpassWithProfileParameters masterpassWithProfileParameters(
     );
   }
 
+  /**
+   * Get parameters for Pivo payment request.
+   *
+   * @param successUrl  The URL the user is redirected after the transaction is handled. The payment itself may still be rejected.
+   * @param failureUrl  The URL the user is redirected after a failure such as an authentication or connectivity error.
+   * @param cancelUrl   The URL the user is redirected after cancelling the transaction (clicking on the cancel button).
+   * @param amount      The amount to pay.
```
Comment (Swie) [inline src/main/java/io/paymenthighway/FormBuilder.java]: If only EURs are support, it should be stated in the comment

File: src/main/java/io/paymenthighway/PaymentAPI.java [hunk 2]
```diff
@@ -130,6 +147,24 @@ public TransactionResponse revertTransaction(UUID transactionId, String amount)
...
+   *
+   * @param transactionId Transaction id
+   * @param referenceNumber Reference number
+   * @param amount Amount to revert
+   * @return Transaction response
+   * @throws HttpResponseException Exception
+   * @throws AuthenticationException Exception
+   * @throws IOException Exception
+   */
+  public TransactionResponse revertPivoTransaction(UUID transactionId, String referenceNumber, String amount) throws IOException {
```
Comment (Swie) [inline src/main/java/io/paymenthighway/PaymentAPI.java]: amount could be Long

File: src/main/java/io/paymenthighway/model/response/PivoTransactionResultResponse.java [hunk 3]
```diff
@@ -0,0 +1,45 @@
+package io.paymenthighway.model.response;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ *  Pivo Transaction Result response POJO.
+ *  Used to find out whether or not an Pivo transaction succeeded.
```
Comment (Swie) [inline src/main/java/io/paymenthighway/model/response/PivoTransactionResultResponse.java]: a

File: src/main/java/io/paymenthighway/model/response/PivoTransactionResultResponse.java [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+    }
+
+    public String getPaymentType() {
+        return paymentType;
+    }
+
+    public String getReferenceNumber() {
+        return referenceNumber;
+    }
+}
```
Comment (Swie) [inline src/main/java/io/paymenthighway/model/response/PivoTransactionResultResponse.java]: no new line

Comment (Kostersson) [reply src/main/java/io/paymenthighway/model/response/PivoTransactionResultResponse.java]: fixed

**Pull request merged**